examples/vue/index.html

```diff
@@ -0,0 +1,115 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Bugsnag: Plain JS Example</title>
+    <script src="../../src/bugsnag.js" data-apikey="API-KEY-GOES-HERE"></script>
+    <script src="./node_modules/vue/dist/vue.js"></script>
+  </head>
+  <style>
+    body {
+      font-family: 'Menlo', monospace; line-height: 1.7;
+      color: #444; background-color: #fafafa;
+      max-width: 600px; margin: 0 auto; padding: 20px;
+    }
+  </style>
+  <body>
+    <h1>Bugsnag: Vue.js Example</h1>
+
+    <!-- The HTML structure for this very basic Vue.js app -->
+    <div id="app">
+      <p>
+        <button v-on:click="triggerRenderError">Throw an error during component render()</button>
+        <span v-if="doARenderError">{{ doARenderError.non.existent.property }}</span>
+      </p>
+      <bad-button></bad-button>
+      <p>
+        <button v-on:click="triggerNextTickError">Throw an error during Vue.nextTick()</button>
+      </p>
+      <p>
+        <button v-on:click="triggerWatchError">Trigger a watch error</button>
+      </p>
+    </div>
+
+    <!-- A component that will trigger a render() error when clicked -->
+    <script type="text/x-template" id="bad-button-template">
+      <p>
+        <button v-on:click="triggerRenderError">Throw an error during component render() (child component)</button>
+        <span v-if="doARenderError">{{ doARenderError.non.existent.property }}</span>
+      </p>
+    </script>
+
+    <script>
+
+      // Provide an error handler to Vue.js. Any lifecycle errors that previously would have
+      // been caught and logged by Vue.js can be sent on to Bugsnag here. This error handler
+      // completely replaces the default behaviour, which means if you want to see the error
+      // in the console it'll need to be logged.
+      Vue.config.errorHandler = function (err, vm, info) {
+        Bugsnag.notifyException(err, {
```

Comment:
If the metadata is nested, the top level values are used as tab names on the dashboard, otherwise anything in here is placed in a tab named "Custom". This data could instead be grouped in a "Vue" tab.

```diff
    Bugsnag.notifyException(err, {
+     vue: {
+       errorInfo: info,
-       vueErrorInfo: info,
        component: vm ? vm.$options.name || vm.$options._componentTag || 'app root' : undefined,
        props: vm ? vm.$options.propsData : undefined
+     }
    })
```